document.go

```diff
@@ -67,7 +67,7 @@ type NumericField interface {
 }
 
 type DateTimeField interface {
-	DateTime() (time.Time, error)
+	DateTime() (time.Time, string, error)
```

Comment:
Rather than changing the existing API within the interface, add a separate API to obtain the string version - 
   
   DateTimeStr() (string, error)

Reply:
Actually, this is ok I think.